src/aero_kernel/src/drivers/drm/mod.rs

```diff
@@ -1093,8 +1093,8 @@ fn make_dmt_modes(max_width: u16, max_height: u16) -> Vec<DrmModeInfo> {
 
     // Sort the modes by display size:
     result.sort_by(|e, f| {
-        (e.hdisplay * e.vdisplay)
-            .partial_cmp(&(f.hdisplay * f.vdisplay))
+        ((e.hdisplay * e.vdisplay) as usize)
```

Comment:
You probably meant to do `(e.hdisplay as usize) * (e.vdisplay as usize)`. Since, that would still overflow.

Reply:
yeah, i just noticed that while i was working on a seperate bug.